CalibCalorimetry/HcalTPGAlgos/src/HcaluLUTTPGCoder.cc

```diff
@@ -99,7 +102,8 @@ void HcaluLUTTPGCoder::init(const HcalTopology* top, const HcalTimeSlew* delay)
   nHFEta_ = (lastHFEta_ - firstHFEta_ + 1);
   maxDepthHF_ = topo_->maxDepth(HcalForward);
   sizeHF_ = 2 * nHFEta_ * nFi_ * maxDepthHF_;
-  size_t nluts = (size_t)(sizeHB_ + sizeHE_ + sizeHF_ + 1);
+  sizeZDC_ = 2 * 9;
+  size_t nluts = (size_t)(sizeHB_ + sizeHE_ + sizeHF_ + sizeZDC_ * 2 + 1);
```

Comment:
That's correct. These are two different factors of two. One is for the two sides of the ZDC and the other is because each ZDC channel gets two LUTs, one regular input LUT and one LUT that is used for OOTPU subtraction.